bundles/org.eclipse.passage.loc.api/src/org/eclipse/passage/loc/internal/api/workspace/ResourceHandle.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2021 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.loc.internal.api.workspace;
+
+public interface ResourceHandle {
+
+	/**
+	 * Tell a client where the content is hosted (path to a file, for instance)
+	 */
+	String info();
```

Comment:
location?

I think `info` assumes something human readable

Reply:
It actually is. A handle, technically, can cover any strange thing without formalization, and here it reports how some readable identifier. For a local file it is, obviousely, absolute path.